catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/RemoveCommandTest.java

```diff
@@ -13,67 +13,93 @@
  */
 package org.codice.ddf.commands.catalog;
 
+import static java.util.stream.Collectors.toList;
 import static org.hamcrest.Matchers.containsString;
 import static org.junit.Assert.assertThat;
+import static org.mockito.Matchers.isA;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
 
 import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Set;
 import java.util.UUID;
+import java.util.stream.Stream;
 
 import org.junit.Before;
 import org.junit.Test;
+import org.mockito.stubbing.Answer;
 
+import com.google.common.collect.ImmutableList;
+
+import ddf.catalog.CatalogFramework;
 import ddf.catalog.cache.SolrCacheMBean;
 import ddf.catalog.data.Metacard;
+import ddf.catalog.data.Result;
 import ddf.catalog.data.impl.MetacardImpl;
+import ddf.catalog.data.impl.ResultImpl;
+import ddf.catalog.operation.DeleteRequest;
+import ddf.catalog.operation.DeleteResponse;
+import ddf.catalog.operation.QueryRequest;
+import ddf.catalog.operation.QueryResponse;
 
 public class RemoveCommandTest extends ConsoleOutputCommon {
 
-    private List<Metacard> metacardList = getMetacardList(5);
+    private String filter;
+
+    private Set<String> ids;
+
+    private List<Metacard> metacardList;
+
+    private CatalogFramework catalogFrameworkMock;
+
+    private RemoveCommand removeCommand;
+
+    private SolrCacheMBean mbeanMock;
+
+    private QueryResponse queryResponse;
+
+    private DeleteResponse deleteResponse;
 
     @Before
     public void setup() {
-        metacardList = getMetacardList(5);
-    }
+        metacardList = populateMetacardList(5);
 
-    @Test
-    public void testSingleItemList() throws Exception {
-        final SolrCacheMBean mbean = mock(SolrCacheMBean.class);
-        RemoveCommand removeCommand = new RemoveCommand() {
+        mbeanMock = mock(SolrCacheMBean.class);
```

Comment:
✏️  We may want to give this variable a better name, e.g., `solrCacheMBeanMock` or just `solrCacheMock`.